packages/common/src/search/types/types.ts

```diff
@@ -1,205 +1,5 @@
-  modified?: IDateRange<number>;
-  orgid?: IMatchOptions;
-  owner?: IMatchOptions;
-  tags?: IMatchOptions;
-  term?: string;
-  title?: IMatchOptions;
-  type?: IMatchOptions;
-  typekeywords?: IMatchOptions;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  [key: string]: any;
-  subFilters?: IContentFilter[];
-}
-
-// Short-cut strings for `type`
-export interface IWellKnownContentFilters {
-  $apps: IContentFilterDefinition[];
-  $dashboard: IContentFilterDefinition[];
-  $dataset: IContentFilterDefinition[];
-  $experience: IContentFilterDefinition[];
-  $site: IContentFilterDefinition[];
-  $storymap: IContentFilterDefinition[];
-  $initiative: IContentFilterDefinition[];
-  $document: IContentFilterDefinition[];
-}
-
-// Allows type-safe query "short-cuts"
-export type NamedContentFilter = keyof IWellKnownContentFilters;
-
-export interface IUserFilterDefinition {
-  created?: IDateRange<number> | IRelativeDate;
-  disabled?: boolean;
-  email?: string | string[] | IMatchOptions;
-  firstname?: string | string[] | IMatchOptions;
-  fullname?: string | string[] | IMatchOptions;
-  group?: string | string[] | IMatchOptions;
-  lastlogin?: IDateRange<number> | IRelativeDate;
-  lastname?: string | string[] | IMatchOptions;
-  modified?: IDateRange<number> | IRelativeDate;
-  role?: string | string[] | IMatchOptions;
-  term?: string;
-  userlicensetype?: string | string[] | IMatchOptions;
-  username?: string | string[] | IMatchOptions;
-  emailstatus?: string | string[] | IMatchOptions;
-  provider?: string | string[] | IMatchOptions;
-}
-
-/**
- * These properties must all be passed through
- * so they are sent in separate fields and *not*
- * serialized into a q
- * Also `joined` does not take the same format as
- * dates in `q`
- */
-export interface IGroupMemberFilterDefinition {
-  memberType?: "admin" | "member";
-  joined?: IDateRange<number> | IRelativeDate;
-  name?: string | string[] | IMatchOptions;
-  orgIds?: string | string[] | IMatchOptions;
-}
-
-export interface IGroupFilterDefinition {
-  access?: string | string[] | IMatchOptions;
-  created?: IDateRange<number> | IRelativeDate;
-  id?: string | string[] | IMatchOptions;
-  isInvitationOnly?: boolean;
-  modified?: IDateRange<number> | IRelativeDate;
-  orgid?: string | string[] | IMatchOptions;
-  owner?: string | string[] | IMatchOptions;
-  searchUserAccess?: "groupMember";
-  searchUserName?: string;
-  tags?: string | string[] | IMatchOptions;
-  term?: string;
-  title?: string | string[] | IMatchOptions;
-  typekeywords?: string | string[] | IMatchOptions;
-  isopendata?: boolean;
-}
-
-export interface IEventFilterDefinition {
-  created?: IDateRange<number> | IRelativeDate;
-  modified?: IDateRange<number> | IRelativeDate;
-  orgid?: string | string[] | IMatchOptions;
-  title?: string | string[] | IMatchOptions;
-}
+// import { UserSession } from "@esri/arcgis-rest-auth";
+// import { IHubContent, AccessLevel } from "../..";
```

Comment:
more commented code that could be deleted

Reply:
removed